cmd/kyverno/localDockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:alpine
+FROM golang@sha256:992d5fea982526ce265a0631a391e3c94694f4d15190fd170f35d91b2e6cb0ba
```

Comment:
This supposed to be `golang:alpine@sha256:e4dcdac3ed37d8c2b3b8bcef2909573b2ad9c2ab53ba53c608909e8b89ccee36` otherwise apk add commands gets failed. i will raise a PR to fix this.